### PR TITLE
Fix typographical errors

### PR DESCRIPTION
**Pull request by gfvcastro**

### CODE REVIEW

File: lib/mix/tasks/hex.organization.ex [hunk 1]
```diff
@@ -7,7 +7,7 @@ defmodule Mix.Tasks.Hex.Organization do
   Manages the list of authorized Hex.pm organizations.
 
   Organizations is a feature of Hex.pm to host and manage private packages. See
-  https://hex.pm/docs/private for more information.
+  `https://hex.pm/docs/private` for more information.
```
Comment (ericmj) [inline lib/mix/tasks/hex.organization.ex]: We shouldn't add this for links since markdown will not render them as links.

Comment (ericmj) [reply lib/mix/tasks/hex.organization.ex]: Seems like earmark doesn't support automatic link creation even with github flavoured enabled. Surrounding with <> angel brackets should work though.

Comment (gfvcastro) [reply lib/mix/tasks/hex.organization.ex]: Works with `<>`. Thank you!

**Comment by ericmj** [comment]
:heart:

**Pull request merged**